src/gameHandlerClass.ts

```diff
@@ -35,4 +35,15 @@ export class GameHandler {
       ? this._playerTwo
       : this._playerOne;
   }
+
+  canIUseMango(): Boolean {
```

Comment:
**canUseMango** is enough :)

Reply:
well, this whole method is probably not necessary :P 
But it might be, if it would be used somewhere outside of gamehandler 🤔

Reply:
I think it will actually be needed to disable button "use Mango"